package/Dockerfile

```diff
@@ -110,8 +110,8 @@ RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
     chmod +x /usr/bin/tini /usr/bin/telemetry /usr/bin/k3s /usr/bin/kubectl && \
     mkdir -p /var/lib/rancher-data/driver-metadata
 
-ENV CATTLE_UI_VERSION 2.6.1
-ENV CATTLE_DASHBOARD_UI_VERSION v2.6.1
+ENV CATTLE_UI_VERSION 2.6.1-rc1
+ENV CATTLE_DASHBOARD_UI_VERSION v2.6.1-rc1
```

Comment:
Should these be 2.6.3-rc1?

Reply:
Yes you are correct. I have updated those with my most recent changes.